lib/ansible/modules/network/icx/icx_command.py

```diff
@@ -0,0 +1,240 @@
+        then all conditionals in the wait_for must be satisfied.  If
+        the value is set to C(any) then only one of the values must be
+        satisfied.
+    type: str
+    default: all
+    choices: ['any', 'all']
+  retries:
+    description:
+      - Specifies the number of times a command should by tried
+        before it is considered failed. The command is run on the
+        target device every retry and evaluated against the
+        I(wait_for) conditions.
+    type: int
+    default: 10
+  interval:
+    description:
+      - Configures the interval in seconds to wait between retries
+        of the command. If the command does not pass the specified
+        conditions, the interval indicates how long to wait before
+        trying the command again.
+    type: int
+    default: 1
+"""
+
+EXAMPLES = """
+tasks:
+  - name: run show version on remote devices
+    icx_command:
+      commands: show version
+
+  - name: run show version and check to see if output contains ICX
+    icx_command:
+      commands: show version
+      wait_for: result[0] contains ICX
+
+  - name: run multiple commands on remote nodes
+    icx_command:
+      commands:
+        - show version
+        - show interfaces
+
+  - name: run multiple commands and evaluate the output
+    icx_command:
+      commands:
+        - show version
+        - show interfaces
+      wait_for:
+        - result[0] contains ICX
+        - result[1] contains GigabitEthernet1/1/1
+  - name: run commands that require answering a prompt
+    icx_command:
+      commands:
+        - command: 'service password-encryption sha1'
+          prompt: 'Warning: Moving to higher password-encryption type,.*'
+          answer: 'y'
+  - name: run commands that require answering multiple prompt
+    icx_command:
+      commands:
+        - command: 'username qqq password qqq'
+          prompt:
+            - 'User already exists. Do you want to modify:.*'
+            - 'To modify or remove user, enter current password:'
+          answer:
+            - 'y'
+            - 'qqq\\\r'
+          check_all: True
+          newline: False
+"""
+
+RETURN = """
+stdout:
+  description: The set of responses from the commands
+  returned: always apart from low level errors
+  type: list
+  sample: ['...', '...']
+stdout_lines:
+  description: The value of stdout split into a list
+  returned: always apart from low level errors
+  type: list
+  sample: [['...', '...'], ['...'], ['...']]
+failed_conditions:
+  description: The list of conditionals that have failed
+  returned: failed
+  type: list
+  sample: ['...', '...']
+"""
+
+
+import re
+import time
+from ansible.module_utils.network.icx.icx import run_commands
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.network.common.utils import ComplexList
+from ansible.module_utils.network.common.parsing import Conditional
+from ansible.module_utils.six import string_types
+
+
+def to_lines(stdout):
```

Comment:
resue the function here https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/network/common/utils.py#L73 to avoid code duplication